File: Day Two/JavaScript/LevelFour/Tasks.js
function adds(number , numberOne){
    return number + numberOne
    }


function isEven(number){
    even = false
    if (number % 2 == 0){
        even = true
    }
    return even
}


function square(number){
    return number**2
}


function celsius_to_fahrenheit(celsius){
    answer = (celsius * 9/5) + 32
    return answer
    }


function isPrime(number){
    for (let count = 2; count <= number/2; count++){
        if (number % count == 0){
            return false
        }
      }
    return true
}


function largestOfThree(number , numberOne , numberTwo){
    largest = number
    if(largest < numberOne){
        largest = numberOne
    }
    if(largest < numberTwo){
        largest = numberTwo   
    }
    return largest 
    }


function simpleInterest(principal , rate , time){
    interest = (principal * rate * time) / 100
    return interest
    }


function areaOfRectangle(length , width){
    area = length * width
    return area
    }


function reverseNumber(number){
    num = 0
    while (number > 0){
        each = number % 10
        num = (num * 10) + each
        number /= 10
    }
    return num
    }

console.log(adds(2 , 6))
console.log(isEven(5))
console.log(square(4))
console.log(celsius_to_fahrenheit(100))
console.log(isPrime(4))
console.log(largestOfThree(90 , 27 , 24))
console.log(simpleInterest(1500 , 50 , 5))
console.log(areaOfRectangle(4 , 6))
console.log(reverseNumber(12345))

File: Day Two/Java/LevelFour/Tasks.java
public class Tasks {
        public static int add(int number , int numberOne) {
                return number + numberOne;
        }

            public static boolean isEven(int number) {
                boolean even = false;
                if (number % 2 == 0){
                        even = true;
                }
                return even;
             }   

                 public static int square(int number) {
                      return number * number;
                  }   

                 public static double fahrenheit(int celsius) {
                    double answer = (celsius * 9/5) + 32;
                      return answer;
                  }  

            public static boolean isPrime(int number) {
                boolean even = false;
                for (int count = 2; count <= number/2; count++){
                     if (number % count != 0){
                        even = true;
                    }
                }
                return even;
             }    

                 public static int largestOfThree(int number , int numberOne , int numberTwo) {
                           int largest = number;
                    if(largest < numberOne){
                            largest = numberOne;
                        }
                    if(largest < numberTwo){
                        largest = numberTwo;
                    }
                    return largest;
            }

                 public static double simpleInterest(double principal , double rate , double time) {
                    double interest = (principal * rate * time) / 100;
                      return interest;
                  }  

                 public static int area(int length , int width) {
                      return length * width;
                  }   


            public static int reverseNumber(int number) {  
                        int reverse = 0;
                    while (number > 0){
                        int each = number % 10;
                            reverse = (reverse * 10) + each;
                        number /= 10;
                    }
                          return reverse;
                    }

                    public static void main(String[] args) {
                            System.out.println(add(5 , 6));
                            System.out.println(isEven(3));
                            System.out.println(square(4));
                            System.out.println(fahrenheit(100));
                            System.out.println(isPrime(4));
                            System.out.println(largestOfThree(23 , 17 , 6));
                            System.out.println(simpleInterest(1500 , 50 , 5));
                            System.out.println(area(4 , 5));
                            System.out.println(reverseNumber(12345));
                    }

}










